src/core/agent/services/identifierService.types.ts

```diff
@@ -16,15 +15,8 @@ interface IdentifierShortDetails {
   isPending: boolean;
 }
 
-type GetIdentifierResult =
-  | { type: IdentifierType.KERI; result: KERIDetails }
-  | { type: IdentifierType.KEY; result: DIDDetails };
+type GetIdentifierResult = { type: IdentifierType.KERI; result: KERIDetails };
```

Comment:
We can remove this too - just return `KERIDetails` but rename it to `IdentifierDetails`